feature/amazon/src/test/java/com/revenuecat/purchases/amazon/AmazonOfferingsTest.kt

```diff
@@ -0,0 +1,18 @@
+//  Purchases
+//
+//  Copyright © 2019 RevenueCat, Inc. All rights reserved.
+//
+
+package com.revenuecat.purchases.amazon
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+//@RunWith(AndroidJUnit4::class)
```

Comment:
i started writing these but felt like it would kinda be wasted time without AmazonStoreProduct work done..